src/get_games.py

```diff
@@ -44,16 +44,22 @@ def save_to_file(game,
 
     """
     filename = os.path.join(save_dir, save_file + '.pgn')
-    with open(filename, 'w') as file:
-        file.write("".join(game))
+    if os.path.exists(filename):
```

Comment:
Append mode already creates a file if it does not exists.

Reply:
Sorted!